src/keystore.c

```diff
@@ -427,6 +427,20 @@ static bool _get_xprv(const uint32_t* keypath, const size_t keypath_len, struct
     return true;
 }
 
+bool get_root_fingerprint(char* fingerprint, int fingerprint_len)
```

Comment:
you can drop `fingerprint_len` and state in the docstring that the passed buffer must be 4 bytes long. This is what we usually do.

Alternatively, use `size_t fingerprint_len`, and Abort() if it isn't `4`. This might make it a bit safer for the programmer, but is technically not needed as it's a constant `4` anyway.